tests/jenkins/TestPromoteArtifacts.groovy

```diff
@@ -71,25 +72,51 @@ class TestPromoteArtifacts extends BuildPipelineTest {
             return [stdout: "zip_dummy_artifact_1.1.0.zip", exitValue: 0]
         }
 
-        Path source = Path.of("tests/data/opensearch-build-1.1.0.yml");
-        target = Path.of("artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/manifest.yml");
+        targetOpenSearch = copy(
+            "tests/data/opensearch-build-1.1.0.yml", 
```

Comment:
Not familiar with code SDE, but yml is 1.2. Here code refer to 1.1, below code refer to 1.3. Should these version match?

Reply:
For the test it doesn't matter I think, but maybe best if they did to avoid confusion.